src/App.jsx

```diff
@@ -17,6 +20,10 @@ return (
     <Route path="/" element={<Articles/>}/>
     <Route path="/articles" element={<Articles/>}/>
     <Route path="/view-article/:article_id" element={<ViewArticle/>}/>
+    <Route path="/topics" element={<Topics/>}/>
+    <Route path="/topics/coding" element={<Coding/>}/>
```

Comment:
It would be more efficient to consider using a parametric endpoint here to make your code more DRY.